### PR TITLE
#4 edit full text plus replace only trainer

### PR DESCRIPTION
**Pull request by MiguelAngelRL**
Now the trainer can modify the session log and update all the students's log.

### CODE REVIEW

**Comment by MiguelAngelRL** [comment]
Issue #4 solved

File: front/src/pods/trainer/components/session.component.tsx [hunk 1]
```diff
@@ -21,8 +39,27 @@ export const SessionComponent: React.FC<Props> = props => {
         rowsMax={20}
         rowsMin={20}
         className={studentBoard}
-        value={log}
+        disabled={false}
+        // onChange={(e)=>modifyLog="adiso"}
```
Comment (brauliodiez) [inline front/src/pods/trainer/components/session.component.tsx:43]: Remove lechuguitas (commented code)

File: front/src/pods/trainer/components/session.component.tsx [hunk 2]
```diff
@@ -21,8 +39,27 @@ export const SessionComponent: React.FC<Props> = props => {
...
       />
+      <Button
+        variant="contained"
+        color="secondary"
+        className={undoBtn}
+        onClick={() => handleSetSessionContent(log)}
+        // disabled={!trainerText}
```
Comment (brauliodiez) [inline front/src/pods/trainer/components/session.component.tsx:50]: Remove lechuguitas

File: front/src/pods/trainer/components/session.component.tsx [hunk 3]
```diff
@@ -21,8 +39,27 @@ export const SessionComponent: React.FC<Props> = props => {
...
+      >
+        Undo
+      </Button>
+      <Button
+        variant="contained"
+        color="primary"
+        className={sendBtn}
+        onClick={() => handleSendFullContentLog(getFullContent(log))}
+        // disabled={!trainerText}
```
Comment (brauliodiez) [inline front/src/pods/trainer/components/session.component.tsx:59]: Remove lechuguita

File: front/src/pods/trainer/components/session.styles.ts [hunk 4]
```diff
@@ -21,3 +21,12 @@ export const studentBoard = css`
...
   font-family: ${typography.fontFamily};
 `;
+
+export const sendBtn = css`
+  width: 50%;
+`;
+
+export const undoBtn = css`
+  width: 50%;
+  color: red;
```
Comment (brauliodiez) [inline front/src/pods/trainer/components/session.styles.ts:31]: Rather use material UI varian primary / secondary or theme options

File: front/src/pods/trainer/components/session.component.tsx (hunk 2 above)
Comment (MiguelAngelRL) [reply front/src/pods/trainer/components/session.component.tsx]: Done

File: front/src/pods/trainer/components/session.component.tsx (hunk 3 above)
Comment (MiguelAngelRL) [reply front/src/pods/trainer/components/session.component.tsx]: Done

File: front/src/pods/trainer/components/session.styles.ts (hunk 4 above)
Comment (MiguelAngelRL) [reply front/src/pods/trainer/components/session.styles.ts:31]: Done

File: front/src/pods/trainer/components/session.component.tsx (hunk 1 above)
Comment (MiguelAngelRL) [reply front/src/pods/trainer/components/session.component.tsx]: Done

**Pull request merged**